### ISSUE TITLE
Missing WhiteLight dependency

### ISSUE THREAD

**Issue by zezic** [stoermelder/vcvrack-packone#45]
Can't compile because of missing WhiteLight:
```
g++ -std=c++11 -Wsuggest-override -fPIC -I/home/zezic/Documents/Soft/Audio/Rack-SDK/include -I/home/zezic/Documents/Soft/Audio/Rack-SDK/dep/include -MMD -MP -g -O3 -march=nocona -funsafe-math-optimizations -Wall -Wextra -Wno-unused-parameter -DARCH_LIN -c -o build/src/ReMove.cpp.o src/ReMove.cpp          
src/ReMove.cpp: In constructor ‘ReMove::ReMoveWidget::ReMoveWidget(ReMove::ReMoveModule*)’:
src/ReMove.cpp:1108:48: error: ‘WhiteLight’ was not declared in this scope
         addChild(createLightCentered<TinyLight<WhiteLight>>(Vec(19.5f, 113.8f), module, ReMoveModule::SEQ_LIGHT + 0));
                                                ^~~~~~~~~~
```

**Comment by stoermelder** [comment]
WhiteLight is included in Rack SDK 1.1.3.

**Comment by zezic** [comment]
Ah, thank you! My bad.
